tests/checksum/reset_all_benchmarks.sh

```diff
@@ -231,10 +231,22 @@ then
         || echo "ctest command failed, maybe just because checksums are different. Keep going"
     cd $checksum_dir
     ./checksumAPI.py --reset-benchmark \
-                     --file_name ${build_dir}/bin/diags/hdf5 \
+                     --file_name ${build_dir}/bin/adaptive_time_step.1Rank \
                      --test-name adaptive_time_step.1Rank
 fi
 
+# radiation_reactions.1Rank
+if [[ $all_tests = true ]] || [[ $one_test_name = "radiation_reactions.1Rank" ]]
```

Comment:
Is this usual plural, i.e. radiation reactions vs. radiation reaction? If not, we could use singular in the test name etc.

Reply:
Ah, this was not on purpose, let's use singular throughout the code, will fix it